collections-csharp/HashtableInCSharp/Tests/HashtableInCSharpTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿using HashtableInCSharp;
+using System.Collections;
+
+namespace Tests
+{
+    public class HashtableInCSharpTest
+    {
+        private readonly HashtableDemo _hashtableDemo;
+        private readonly Hashtable _users;
+        public HashtableInCSharpTest()
+        {
+            _hashtableDemo = new HashtableDemo();
+            _users = _hashtableDemo.AddToHashTable(new Hashtable());
+        }
+        [Fact]
+        public void GivenAnEmptyHashtable_WhenAddingToAHashtable_ThenReturnsFilledEmptyHashTable()
```

Comment:
This test isn't actually adding anything to a HashTable, it should create an empty HashTable and then call the "AddToHashTable" method and finally verify that something was added.